system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/dataspaceconnector/test/e2e/EndToEndTransferTest.java

```diff
@@ -198,6 +202,25 @@ void httpPullDataTransfer() {
         });
     }
 
+    private void createAssetAndContractDefinitionOnProvider() {
+        var assetId = "asset-id";
+        createAsset(PROVIDER_CONTROL_PLANE, assetId);
+        var policyId = createPolicy(assetId, PROVIDER_CONTROL_PLANE);
+        createContractDefinition(policyId, PROVIDER_CONTROL_PLANE);
+    }
+
+    private Catalog getCatalog(URI provider, URI instance) {
+        return given()
+                .baseUri(instance.toString())
+                .contentType(JSON)
+                .when()
+                .queryParam("providerUrl", provider + IDS_PATH + "/data")
+                .get("/api/catalog")
```

Comment:
Is this path correct? Shouldn't it be `/api/v1/data/catalog`?

Reply:
No, because since no `data` mapping is configured, so data management is exposed on the `default` mapping, that is `/api` as configuration (row 75)